values-openshift.yaml

```diff
@@ -1,3 +1,6 @@
+route:
```

Comment:
There are few keys are not defined. I suggest to add these in the values-openshift.yaml
```yaml
route:
  annotations: {}
  # Add custom dns URL for your route.
  # Example:  
  # host: kubecost.apps.okd4.example.com
 ```

Reply:
I've changed according to your suggestion.